src/builder.rs

```diff
@@ -73,12 +73,24 @@ impl<'a> RtpPacketBuilder<'a> {
         self
     }
 
+    /// Set 1. whether padding is used and 2. the number of padding bytes to use for this packet
+    pub fn maybe_padding(mut self, padding: Option<u8>) -> Self {
```

Comment:
Padding can be used to align to a 4-byte boundary, it can also be used to align to an encryption cipher's block size which may be larger than 4. As such, we don't limit the actual size of the padding.

Reply:
Ah, good to know